ext/opentelemetry-ext-grpc/src/opentelemetry/ext/grpc/__init__.py

```diff
@@ -16,9 +16,70 @@
 # pylint:disable=import-self
 # pylint:disable=no-name-in-module
 # pylint:disable=relative-beyond-top-level
+from contextlib import contextmanager
 
+import grpc
+from concurrent import futures
 from opentelemetry import trace
 from opentelemetry.ext.grpc.version import __version__
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.auto_instrumentation.instrumentor import BaseInstrumentor
+from opentelemetry.trace import get_tracer
+from wrapt import ObjectProxy
+from wrapt import wrap_function_wrapper as _wrap
+from opentelemetry.ext.grpc.grpcext import intercept_channel, intercept_server
```

Comment:
Should these imports be in alphabetical order?

Reply:
fyi, travis WILL fail if you don't sort your imports, so make sure to call tox lint before pushing changes

Reply:
thanks andrew!